src/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "noImplicitAny": true,
     "noImplicitThis": true,
     "moduleResolution": "Node",
+    "esModuleInterop": true,
```

Comment:
with the addition of SortableJS, typescript would not build without this (i also tried `import * as Sortable`)

Is there any reason we had this set to false?